sdk/eventgrid/azure-eventgrid/tests/eventgrid_preparer.py

```diff
@@ -86,7 +86,6 @@ def remove_resource(self, name, **kwargs):
         if self.is_live:
             group = self._get_resource_group(**kwargs)
             self.client.topics.delete(group.name, name, polling=False)
-
     def _get_resource_group(self, **kwargs):
```

Comment:
seems like we'd want this whitespace?
```suggestion

    def _get_resource_group(self, **kwargs):
```

Reply:
prolly crept in when i rebased